src/lib/dom.generated.d.ts

```diff
@@ -1873,7 +1885,7 @@ interface AbortSignal extends EventTarget {
 declare var AbortSignal: {
     prototype: AbortSignal;
     new(): AbortSignal;
-    // abort(): AbortSignal; - To be re-added in the future
+    abort(reason?: any): AbortSignal;
```

Comment:
This I thought was a part of the automation, but I guess not - might have to change it on the DOM side

Reply:
You mean we might need to add an override?

Reply:
Nah, this was not supposed to happen because it conflicts with the node AbortSignal - but the deploy process didn't take into account the new param - https://github.com/microsoft/TypeScript-DOM-lib-generator/commit/1bba42689ff3b918dba01276a1356957d1f02cd8